okio-files/src/mingwX64Main/kotlin/okio/posix_variant.kt

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
is this the right name for this file?

Reply:
seems fine. i'd probably call it posixVariant.kt. underscores are a bit weird.

Reply:
or variantPosix.kt